srst2/0.2.0/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:xenial as app
+
+# for easy upgrade later. ARG variables only persist at build time
+ARG SRST2_VER=0.2.0
+
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="SRST2"
+LABEL software.version="v0.2.0"
+LABEL description="Short Read Sequence Typing for Bacterial Pathogens"
+LABEL website="https://github.com/katholt/srst2"
+LABEL license="https://github.com/katholt/srst2/blob/master/LICENSE.txt"
+LABEL maintainer="Holly Halstead"
+LABEL maintainer1.email="holly.halstead@doh.wa.gov"
+
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y \
+python2.7 \
+python-pip \
+python-scipy \
+python-biopython \
+make \
+libc6-dev \
+g++ \
+zlib1g-dev \
+build-essential \
+git \
+libx11-dev \
+xutils-dev \
+zlib1g-dev \
+python-pip \
+bowtie2=2.2.6-2 \
+curl \
+libncurses5-dev \
+unzip \
+wget  \
+locate
+
+# download pre-compiled binary; unzip; put binary in /usr/bin; make /data
+RUN curl -O -L https://sourceforge.net/projects/samtools/files/samtools/0.1.18/samtools-0.1.18.tar.bz2 && \
+tar xvfj samtools-0.1.18.tar.bz2 && \
+cd samtools-0.1.18 && \
+make && \
+cp samtools /usr/local/bin && \
+cp bcftools/bcftools /usr/local/bin && \
+cd misc/ &&  \
+cp *.pl maq2sam-long maq2sam-short md5fa md5sum-lite wgsim /usr/local/bin/ && \
+cd
+
+# Install SRST2
+RUN pip install biopython git+https://github.com/katholt/srst2.git@v${SRST2_VER} && \
+mkdir /data
+
+# set working directory
+WORKDIR /data
+
+FROM app as test
+
+RUN srst2 --version && \
+getmlst.py -h
```

Comment:
This is off to a great start! For a lot of bioinformatic tools, help and version simply test if srst2 is in PATH. For this docker container, it would be nice to test that srst2 can actually run.

Could you add all the recommended tests? They look like they're found at https://github.com/katholt/srst2#3---test-that-the-programs-are-installed-properly

And then adding something like the following (running on some actual file) would be wonderful. The following was modified from https://github.com/katholt/srst2/blob/master/example.txt :
```
RUN getmlst.py --species "Escherichia coli#1" && \
wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR024/ERR024070/ERR024070_1.fastq.gz && \
wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR024/ERR024070/ERR024070_2.fastq.gz && \
srst2 --input_pe ERR024070*.fastq.gz --output shigella1 --log --save_scores --mlst_db Escherichia_coli#1.fasta --mlst_definitions ecoli.txt --gene_db ARGannot.fasta && \
ls shigella1__mlst__Escherichia_coli#1__results.txt \
	shigella1__fullgenes__ARGannot__results.txt \
	shigella1__genes__ARGannot__results.txt \
	shigella1__compiledResults.txt \
	shigella1__ERR024070.Escherichia_coli#1.scores \
	shigella1__ERR024070.ARGannot.scores 
```

Reply:
Added!